charts/grafana/ci/with-sidecars-envvaluefrom-values.yaml

```
@@ -0,0 +1,38 @@
+extraObjects:
+  - apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: '{{ include "grafana.fullname" . }}'
+    data:
+      var1: "value1"
+  - apiVersion: v1
+    kind: Secret
+    metadata:
+      name: '{{ include "grafana.fullname" . }}'
+    type: Opaque
+    data:
+      var2: "value2"
```

data in secrets must be base64 encoded

```suggestion
      var2: "dmFsdWUy"
```